src/commands/bundles.js

```diff
@@ -1,3 +1,4 @@
+const console = require('../console');
```

Comment:
Yeah, I had that gut feeling, but I think I was feeling a little lazy. I can rename it to logger.